kvm/src/ioctl_wrap.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit related to the commit message: I would remove the last bullet because the bindings are not common to all architectures.

Reply:
Removed last two bullets.